ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -185,69 +224,116 @@ const POPULATIONS = [
   {
     field: 'gnomad_mito',
     fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
     precision: 3,
-    urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
-    queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+    ...GNOMAD_URL_INFO,
+  },
+  {
+    field: 'gnomad_mito_heteroplasmy',
+    fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+    precision: 3,
+    ...GNOMAD_URL_INFO,
   },
   {
     field: 'helix',
     fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
     precision: 3,
   },
+  {
+    field: 'helix_heteroplasmy',
+    fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+    precision: 3,
+  },
+]
+
+const acDisplay = (populations, popConfig) => ([
+  `${popConfig.fieldTitle}: ${populations[popConfig.field].ac} out of ${populations[popConfig.field].an}`,
+])
+const DETAIL_SECTIONS = [
+  {
+    name: GLOBAL_AF_SECTION,
+    hasDetail: (populations, popConfig) => populations[popConfig.field] && populations[popConfig.field].filter_af &&
+      (populations[popConfig.field].filter_af !== populations[popConfig.field].af),
+    display: (populations, popConfig) => ([
+      `${popConfig.fieldTitle} : ${populations[popConfig.field].af.toPrecision(popConfig.precision || 2)}`,
```

Comment:
I really dislike having `${popConfig.fieldTitle}:` copy pasted into every single `display` function, it makes the maintenance on this nearly impossible. Shared behavior needs to be shared not copy pasted 5 times. If you for some reason don't like the approach I proposed where `display` returns a list of `subfield` and `value` maybe try something like this instead:

```
const DETAIL_SECTIONS = [{
  ...,
  detailFields: ['af' , 'max_hl'], 
} ...]

const FIELD_SUBTITLES = { max_hl: ' max observed heteroplasmy' }
const FIELD_DISPLAY_VALUE = {
  ac: pop => `${pop.ac} out of ${pop.an}}`,
  af: pop, popConfig => pop.af.toPrecision(popConfig.precision || 2),
  ...
}

...
            const population = populations[popConfig.field]
            accDisplay.push(...section.detailFields.filter(field => population[field]).map(field => (
              <Popup.Content key={`${section.name}${field}`}>
                 {`${pop.fieldTitle}${FIELD_SUBTITLES[field] || ''}: ${FIELD_DISPLAY_VALUE[field](population, popConfig)}`}
              </Popup.Content>
            )))
```

Reply:
> If you for some reason don't like the approach I proposed where `display` returns a list of `subfield` and `value` maybe try something like this instead

Your `subtitle` and `value` suggestion looks good. I just didn't understand when I first looked at it.